testing/unittests/metrics/MetricFrameworkTests.cpp

```diff
@@ -348,18 +348,68 @@ class MetricFrameworkTests : public CppUnit::TestFixture
         pHistogram->recordMeasurement(8);
         sumMeasurements += 8;
         checkHistogramBucketResult(pHistogram, sumMeasurements, {0,2,1,1});
+
+
+        //
+        // Test the scaled time histogram. Buckets for 500ms, 1000ms and 2000ms
+        std::vector<BucketDef> timeBuckets = {{"le 500", 500}, {"le 1000", 1000}, {"le 2000", 2000} };
+        std::shared_ptr<ScaledTimeFromCyclesHistogram> pTimeHistogram = std::make_shared<ScaledTimeFromCyclesHistogram>("requests.time", "description", TimeUnits::HISTOGRAM_TIMEUNITS_MS, timeBuckets);
+        int totalDelay = 0;
+
+        //
```

Comment:
There should be at least 1 test that checks that the scaled value reported back is correct.

Reply:
There are tests that do that, see the asserts on totalDelay following each block. I changed each test to check a range +/- 10 to take into account any rounding. Local testing has shown the delay to always be right on, but I felt it best to test the range since execution time measurements is not an exact science.